runtime/src/vote_account.rs

```diff
@@ -261,7 +271,7 @@ impl PartialEq<VoteAccounts> for VoteAccounts {
     }
 }
 
-type VoteAccountsHashMap = HashMap<Pubkey, (/*stake:*/ u64, VoteAccount)>;
+pub type VoteAccountsHashMap = HashMap<Pubkey, (/*stake:*/ u64, VoteAccount)>;
```

Comment:
nit:  `HashMap<Pubkey, (u64 /*stake*/, ArcVoteAccount)>` is still used in `struct VoteAccounts {`, maybe we could replace that there too